### ISSUE TITLE
Dimension blacklist ignores prop mobs

### ISSUE THREAD

**Issue by Immortallis1** [Silentine/GrimoireOfGaia#126]
version 1.6.9.3

Apparently the "prop" mobs such as mimics and vases extend EntityAgeable instead of MobHostileBase or MobHostileAssist. The dimensional blacklist seems to only deny spawn events for the latter two classes so the prop mobs end up spawning all the time even in blacklisted dimensions.

**Comment by Renari** [comment]
As a workaround https://minecraft.curseforge.com/projects/in-control can be used to prevent prop mobs from spawning as well.

```
[
   {
      "mob":[
         "grimoireofgaia:vase",
         "grimoireofgaia:mimic",
         "grimoireofgaia:chest"
      ],
      "dimension":2,
      "result":"deny"
   }
]
```
You can repeat this block in `spawn.json` for each dimension you'd like to blacklist.

**Comment by Silentine** [comment]
Fixed in 1.7.0
